packages/mds-db/schema.ts

```diff
@@ -72,7 +72,7 @@ const COLUMN = Enum(
   'mfgr',
   'migration',
   'mimetype',
-  'min_violoation_dist',
+  'min_violation_dist',
```

Comment:
Do any of these schema changes require a db migration?

Reply:
Yes it will :/

Reply:
Since a migration is required, you'll have to write one, imitating the ones in `mds-db/migration.ts`. Then you'll have to hit one of the writeable endpoints on `mds-agency` to trigger the migration.